packages/react-native-otlp/ios/RNEmbraceOTLP/RNEmbraceOTLP.swift

```diff
@@ -0,0 +1,223 @@
+import React
+import JavaScriptCore
+import Foundation
+import EmbraceIO
+import EmbraceCrash
+import OpenTelemetrySdk
+import EmbraceCommonInternal
+
+class SDKConfig: NSObject {
```

Comment:
similarly here for sharing with core, not sure how easy it would be though given it's at the swift layer

Reply:
can file a follow-up ticket, I think fine for the initial commit